test/integrations/destinations/campaign_manager/processor/data.ts

```diff
@@ -827,4 +827,368 @@ export const data = [
       },
     },
   },
+  {
+    name: 'campaign_manager',
+    description: 'Test 6: Enhanced Conversions with un-hashed data in request payload',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            message: {
+              channel: 'web',
+              context: {
+                app: {
+                  build: '1.0.0',
+                  name: 'RudderLabs JavaScript SDK',
+                  namespace: 'com.rudderlabs.javascript',
+                  version: '1.0.0',
+                },
+                device: {
+                  id: '0572f78fa49c648e',
+                  name: 'generic_x86_arm',
+                  type: 'Android',
+                  model: 'AOSP on IA Emulator',
+                  manufacturer: 'Google',
+                  adTrackingEnabled: true,
+                  advertisingId: '44c97318-9040-4361-8bc7-4eb30f665ca8',
+                },
+                traits: {
+                  email: 'alex@example.com',
+                  phone: '+1-202-555-0146',
+                  firstName: 'John',
```

Comment:
Please refactor the test cases to reduce duplicity please check Klaviyo for reference

Reply:
This will increase the scope of the task. We can take this as a separate task